### ISSUE TITLE
"Pimple warning" when paging content. 

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#5094]
When paging content like `/?&page_entries=2`, I get this notice: 

> Warning: Missing argument 1 for Pimple::{closure}() in /Users/bob/Sites/bolt-origin/vendor/pimple/pimple/lib/Pimple.php on line 122
> Notice: Undefined variable: c in /Users/bob/Sites/bolt-origin/vendor/pimple/pimple/lib/Pimple.php on line 126

See screenshot: 

![screenshot 2016-03-28 15 04 36](https://cloud.githubusercontent.com/assets/1833361/14078513/a8992020-f4f6-11e5-9d99-fc72d30dc15f.png)

Details
-------

 - Relevant Bolt Version: master

**Comment by bobdenotter** [comment]
Partial backtrace, from the Pimple warning: 

```
array:13 [▼
  0 => array:6 [▼
    ”file" => ”/Users/bob/Sites/bolt-origin/src/EventListener/PagerListener.php"
    ”line" => 45
    ”function" => ”{closure}"
    ”class" => ”Pimple"
    ”type" => ”::"
    ”args" => array:1 [▶]
  ]
  1 => array:7 [▼
    ”file" => ”/Users/bob/Sites/bolt-origin/src/EventListener/PagerListener.php"
    ”line" => 45
    ”function" => ”__invoke"
    ”class" => ”Closure"
    ”object" => Closure {#638 ▶}
    ”type" => ”->"
    ”args" => array:1 [▶]
  ]
  2 => array:5 [▼
    ”function" => ”onRequest"
    ”class" => ”Bolt\EventListener\PagerListener"
    ”object" => PagerListener {#1305 ▶}
    ”type" => ”->"
    ”args" => array:3 [▶]
  ]
  3 => array:4 [▼
    ”file" => ”/Users/bob/Sites/bolt-origin/vendor/symfony/event-dispatcher/Debug/WrappedListener.php"
    ”line" => 61
    ”function" => ”call_user_func"
    ”args" => array:4 [▶]
  ]
```

A "fix" for this is to change this in `src/EventListener/PageListener.php`, line 45:

    $manager = $this->managerFactory->__invoke($dummy = null);

But, there must be a cleaner way to do this.

**Comment by rixbeck** [comment]
I'm on cleaning up...

**Comment by bobdenotter** [comment]
Fixed in #5125
